README.md

```diff
@@ -24,7 +26,7 @@ SOURCE_HOME_DIR=$HOME/workarena/software/sources
 
 
 #### Clone Repository ####
-Clone spack repository from GitHub. You can use [official repository](https://github.com/LLNL/spack) from LLNL but I am adding few modifications / workarounds for existing bugs and other systems like Lugano bg-q / vizcluster.
+Clone spack repository from GitHub. You can use [official repository](https://github.com/LLNL/spack) from LLNL. But we use fork here as few modifications / workarounds (for existing bugs on some systems like BBP IV BG-Q, OS X) have been added.
```

Comment:
> You can use [official repository]

It might be confusing to first suggest to clone the official repository and then ask them to clone your fork. I would ask the users to clone this fork to take benefits of a few modifications and workarounds for existing bugs on some systems like ...
And optionally, than can pick the official repository if bleeding edges changes are needed.

Reply:
updated as per comment.

Actually, there might be very few changes between this fork and llnl repository now. Just need to go through those and test it via Jenkins plan on Lugano bg-q and viz cluster (with all compilers).